.github/workflows/build-test-core-win64.yml

```diff
@@ -0,0 +1,81 @@
+# AUTOGENERATED FROM build-test-core-win64.jsonnet DO NOT MODIFY
+name: build-test-core-win64
+"on":
+  workflow_dispatch:
+  workflow_call:
+jobs:
+  job:
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: bash
+        working-directory: checkout
+    steps:
+    - name: Configure git
+      working-directory: ""
+      run: |
+        git config --system core.longpaths true
```

Comment:
please add a comment why we need thise longpaths, and what happens if we don't have
this option (is it the thing that fixes the issues I had with the symlink to .atd files in my Windows POC?)

Reply:
interesting, that's probably it indeed.